packages/components/Avatar/src/Avatar.styling.ts

```diff
@@ -173,43 +194,91 @@ export const stylingSettings: UseStylingOptions<AvatarProps, AvatarSlotProps, Av
   },
 };
 
-function getRingConfig(tokens: AvatarTokens): any {
+function getRingConfig(tokens: AvatarTokens): RingConfig {
```

Comment:
Maybe it'll be good to split this method into two: getRingConfigWin32 and getRingConfigAngroid for better readability?

Reply:
You could create a file that holds stylingUtils.android/win32.ts or something, to minimize code that isn't used in other platforms.

Reply:
Agrees, the split by platform files to clean this code piece for readability.